tools/unit_test_race.sh

```diff
@@ -54,7 +54,7 @@ for pkg in $flaky_tests; do
   max_attempts=3
   attempt=1
   # Set a timeout because some tests may deadlock when they flake.
-  until go test -timeout 2m $VT_GO_PARALLEL $pkg -v -race -count=1; do
+  until go test -timeout 5m $VT_GO_PARALLEL $pkg -v -race -count=1; do
```

Comment:
The timeout was extended for unit tests in https://github.com/vitessio/vitess/pull/15797/files#diff-470746102447655f4af9e1e9c9756444906143d6b70e394d42eff2aa251fa05aR87 but the race one was not (it should have been — I think that work was done in parallel with them getting split).